bool offboard_active
---
bool status 
